fiftyone/core/config.py

```diff
@@ -244,6 +244,27 @@ def __init__(self, d=None):
         self.timezone = self.parse_string(
             d, "timezone", env_var="FIFTYONE_TIMEZONE", default=None
         )
+
+        self.default_parallelization_method = self.parse_string(
+            d,
+            "default_parallelization_method",
+            env_var="FIFTYONE_DEFAULT_PARALLELIZATION_METHOD",
+            default=None,
+        )
+
+        self.default_thread_pool_workers = self.parse_int(
+            d,
+            "default_thread_pool_workers",
+            env_var="FIFTYONE_DEFAULT_THREAD_POOL_WORKERS",
+            default=None,
+        )
+        self.default_process_pool_workers = self.parse_int(
+            d,
+            "default_process_pool_workers",
+            env_var="FIFTYONE_DEFAULT_PROCESS_POOL_WORKERS",
+            default=None,
+        )
```

Comment:
how is this semantically different from max_workers vars a few lines below?

Reply:
We do use them slightly differently:
```
workers = (
                config.default_thread_pool_workers
                or fou.recommend_thread_pool_workers()
            )

if config.max_thread_pool_workers is not None:
    workers = min(workers, config.max_thread_pool_workers)
```

I guess max workers is the maximum of number of workers for threads/processes, so another way for admin to configure this behavior. @j053y

Reply:
Default is the desired number of x" to use max is the limit. When both supplied in the config it's makes less sense. but the max is used when "x" is manually provided at runtime to limit the number if it exists.

Reply:
I see

Reply:
If it's confusing, add a comment for the next person 😄